src/init/features/apphosting/index.ts

```diff
@@ -117,92 +94,103 @@ export async function doSetup(setup: any, projectId: string): Promise<void> {
     return;
   }
 
-  const { build } = await onboardRollout(projectId, location, backendId, {
+  await orchestrateRollout(projectId, location, backendId, {
     source: {
       codebase: {
         branch,
       },
     },
   });
 
-  if (build.state !== "READY") {
-    if (!build.buildLogsUri) {
-      throw new FirebaseError(
-        "Failed to build your app, but failed to get build logs as well. " +
-          "This is an internal error and should be reported",
-      );
-    }
-    throw new FirebaseError(
-      `Failed to build your app. Please inspect the build logs at ${build.buildLogsUri}.`,
-      { children: [build.error] },
-    );
-  }
-
   logSuccess(`Successfully created backend:\n\t${backend.name}`);
   logSuccess(`Your site is now deployed at:\n\thttps://${backend.uri}`);
-  logSuccess(
-    `View the rollout status by running:\n\tfirebase apphosting:backends:get ${backendId} --project ${projectId}`,
-  );
 }
 
-async function promptLocation(projectId: string, locations: string[]): Promise<string> {
-  return (await promptOnce({
-    name: "region",
-    type: "list",
-    default: DEFAULT_REGION,
-    message:
-      "Please select a region " +
-      `(${clc.yellow("info")}: Your region determines where your backend is located):\n`,
-    choices: locations.map((loc) => ({ value: loc })),
-  })) as string;
+/**
+ * Prompts the user for a backend id and verifies that it doesn't match a pre-existing backend.
+ */
+async function promptNonexistentBackend(projectId: string, location: string): Promise<string> {
```

Comment:
*nit: `promptNewBackendId`? Nonexistent might imply we're not about to create a new backend with that ID, but honestly works fine too